doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -700,7 +700,7 @@ imply multiplication:
     julia> (x-1)(x+1)
     ERROR: MethodError: `call` has no method matching call(::Int64, ::Int64)
     Closest candidates are:
-      Union(!Matched::Any...)
+      Union{!Matched::Any...}
```

Comment:
Are you also changing the printing here? If not then this'll make the doctest fail.